pkg/netstore/netstore.go

```diff
@@ -35,6 +35,7 @@ func (s *store) Get(ctx context.Context, mode storage.ModeGet, addr swarm.Addres
 			// request from network
 			data, err := s.retrieval.RetrieveChunk(ctx, addr)
 			if err != nil {
+				//TODO: INVOKE RECOVERY PROCESS, obtain targets thru ctx
```

Comment:
Could you print something here to the console (perhaps just a log-line), such that I can test out that this works manually?

Reply:
i think that we should not have `TODO`s in the form of logger debug lines. it is pretty straightforward that some functionality needs to be implemented and where. please don't merge this sort of stuff into `master`